src/include/distributed/master_metadata_utility.h

```diff
@@ -81,7 +81,7 @@ typedef struct ShardPlacement
 	/* the rest of the fields aren't from pg_dist_placement */
 	char *nodeName;
 	uint32 nodePort;
-	int nodeId;
+	uint32 nodeId;
```

Comment:
Probably beyond scope, but we might want to introduce typedefs for things like ShardId, maybe NodeId.

Reply:
would be great to add as a small tech dept reduction item when anyone has some downtime next release. For now I think keeping it small makes a lot of sense